flex_config/file_sources/file_source.py

```diff
@@ -0,0 +1,36 @@
+from pathlib import Path
+from typing import Any, Dict, Iterable, TextIO, Tuple
+
+from flex_config.config_source import ConfigSource
+
+
+class FileSource(ConfigSource):
```

Comment:
If you inherit from ABC then I think PyCham (or whatever) will complain if anyone tries to instantiate this directly. Then you an use `abstractmethod` too